core/jobs.py

```diff
@@ -1429,11 +1429,16 @@ def _clear_inactive_realtime_layer(cls, latest_created_on_datetime):
 
     @classmethod
     def _kickoff_batch_job(cls):
```

Comment:
Should there be tests that verify the changes in this file?

Reply:
Done.